Sprint-2/Exercicio-7-Resposta-2/class.ts

```diff
@@ -0,0 +1,66 @@
+export default class List {
+    private _list: string[];
+    private _averageValue: number;
+    private _validator: number;
+    constructor(list: string[]) {
+        this._list = list;
+        this._averageValue = 0;
+        this._validator = 0;
+    }
+
+    numericalList(){
+        /**
+         * This method validates if the list is numeric.
+         * @param _validator - has predefined value as 0
+         * 
+         * if the parseFloat function fails to make it a number and it is not zero
+         * change value of _validator to -1, because have an non numeric value on list
+         * 
+         * @returns 0 or -1
+         */
+        this._list.forEach((arrayItem) => {
+            (parseFloat(arrayItem) || arrayItem == "0") ? parseFloat(arrayItem) : this._validator = -1;
+        });
+        return this._validator;
+    }
+    findBiggerSmallerAndAverageValue(){
+        /**
+         * This method finds the largest, smallest, and average value, adds it to an array, 
+         * and returns the array.
+         * 
+         * @param _list - it's a numerical list to work with.
+         * @param biggerValue - gets the first value of the list
+         * @param smallerValue - gets the first value of the list
+         * @param value - get each value from the list to compare
+         * @param _averageValue - add each value to itself
+         * @param array - receive biggerValue, smallerValue and _averageValue
+         * 
+         * The biggerValue compares if the value is greater than it, if it is, 
+         * it will become the value.
+         * The smallerValue compares if the value is smaller than it, if it is, 
+         * it will become the value.
```

Comment:
Se não for utilizar a variavel index, não precisa passa-la como parametro da função 
 basta um `(arrayItem) => `